subsys/bluetooth/audio/Kconfig.pacs

```diff
@@ -83,9 +104,30 @@ config BT_PAC_SRC_LOC_WRITEABLE
 	  This option enables support for clients to write to the Source PAC
 	  Location Characteristic.
 
+config BT_PAC_SRC_LOC_NOTIFIABLE
+	bool "Source Audio Location Notifiable Support"
+	depends on BT_PAC_SRC_LOC
+	help
+	  This option enables support for clients to be notified on the Source
+	  Audio Location Characteristic.
+
 endif # BT_PAC_SRC
 
 config BT_PACS
 	def_bool BT_PAC_SNK || BT_PAC_SRC
 
+config BT_PACS_SUPPORTED_CONTEXT_NOTIFIABLE
+	bool "Supported Audio Context Notifiable Support"
+	depends on BT_PACS
+	help
+	  This option enables support for clients to be notified on the
+	  Supported Audio Contexts Characteristic.
+
+config BT_PACS_AVAILABLE_CONTEXT_NOTIFIABLE
```

Comment:
Isn't notify in Available Contexts characteristics mandatory?

Reply:
Yes, you are right. That "None" camouflaged itself as a "Notify" very well for me